cli/build.rs

```diff
@@ -63,6 +63,7 @@ fn get_git_hash() -> Option<String> {
             "--color=never",
             "log",
             "--no-graph",
+            "--limit=1",
```

Comment:
nit: I don't think picking arbitrary parent (in index order) is correct, but it's practically okay. We have a similar problem in `git` code path.

@ilyagr might have some opinion. iirc, he had a patch to improve version string.

Reply:
I had also considered using `heads(@ & merges() | @-)` to use `@` instead of `@-` if it's a merge, not sure if that would be any better.